aiida_pseudo/groups/mixins/cutoffs.py

```diff
@@ -255,6 +255,11 @@ def get_cutoffs_unit(self, stringency: str = None) -> str:
         try:
             return self._get_cutoffs_unit_dict()[stringency]
         except KeyError as exception:
+            # Workaround to deal with pseudo families installed in v0.5.0 - Set default unit in case it is not in extras
+            if stringency in self.get_cutoff_stringencies():
+                self.set_cutoffs(self.get_cutoffs(stringency), stringency, unit=self.DEFAULT_UNIT)
```

Comment:
The cutoffs should already be set here and we are merely adding the unit, which should be `eV`. I would make this more explicit and actually hardcode `eV` instead of `self.DEFAULT_UNIT` in the case this changes in the future. Additionally, maybe we should just do
```
cutoffs_unit_dict = self._get_cutoffs_unit_dict()
cutoffs_unit_dict[stringency] = 'eV'
self.set_extra(self._key_cutoffs_unit, cutoffs_unit_dict)
```
It is maybe more lines, but it is showing more explicitly what we are doing. We don't need to set the cutoffs, just the unit, which was missing